airflow/cli/commands/dag_command.py

```diff
@@ -497,6 +524,56 @@ def dag_test(args, session=None):
             print(dot_graph.source)
 
 
+def _run_task(ti: TaskInstance, session):
+    """
+    Run a single task instance, and push result to Xcom for downstream tasks. Bypasses a lot of
+    extra steps used in `task.run` to keep our local running as fast as possible
+    Args:
+        ti: TaskInstance to run
+    """
+    log.info("*****************************************************")
+    log.info("Running task %s", ti.task_id)
+    try:
+        ti._run_raw_task(session=session)
+        log.info("%s ran successfully!", ti.task_id)
```

Comment:
You'll want to customize this message a bit for mapped tasks, else it just says the same tasks id is running many times. Not very helpful

Reply:
@ashb what would a good message for this look like? And how would I know which mapped task is running?

Reply:
could just add ti.map_index

Reply:
if ti.map_index > 0 it's a mapped instance

Reply:
ok added that